README.adoc

```diff
@@ -2,10 +2,9 @@
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 
-https://travis-ci.org/se-edu/addressbook-level4[image:https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master[Build Status]]
-https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/se-edu/addressbook-level4?branch=master[image:https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master[Coverage Status]]
-https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&utm_medium=referral&utm_content=se-edu/addressbook-level4&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a[Codacy Badge]]
+https://travis-ci.org/CS2103AUG2017-W14-B3/main[image:https://travis-ci.org/CS2103AUG2017-W14-B3/main.svg?branch=master[Build Status]]
+https://ci.appveyor.com/project/danielbrzn/main[image:https://ci.appveyor.com/api/projects/status/5dietu4dyky37v1w?svg=true[Build status]]
```

Comment:
Why is it pointing to your own account and not the project repo?

Reply:
After checking AppVeyor, it is pointing to the correct repository. I was the user who set up AppVeyor on the project repo so that explains why my username appears in the link.